sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcore.hpp

```diff
@@ -16,25 +16,28 @@ namespace oneapi {
 namespace experimental {
 namespace matrix {
 
-enum class matrix_use { a, b, accumulator };
+enum class matrix_use { a, b, accumulator, unnecessary };
 
-enum class matrix_layout { row_major, col_major, packed_a, packed_b };
+enum class layout { row_major, col_major, packed_a, packed_b, none};
```

Comment:
Also, if are calling it "layout", we should remove matrix from "matrix_use" as well, right?

Reply:
I agree. It could be worth having a final conversation about the semantics of layout/use before deciding on the naming. I wanted to focus on making sure we are on the same page about the interfaces first though.